Dockerfile.windows

```diff
@@ -116,6 +116,7 @@ RUN cmake -G "'Visual Studio 16 2019'" -DCMAKE_BUILD_TYPE=RELWITHDEBINFO `
     -DFLB_METRICS=On `
     -DFLB_WASM=Off `
     -DFLB_WAMRC=Off `
+    -DFLB_KAFKA=OFF `
```

Comment:
Does this mean we don't need `-DFLB_(IN|OUT)_KAFKA=Off` anymore?

Also, nit: `OFF` -> `Off`